### ISSUE TITLE
Wrong css includes

### ISSUE THREAD

**Issue by silviucpp** [Mermade/shins#75]
Hello,

I'm using master and when I'm generating the output with:

```sh
node shins.js --customcss --css pub/css/tradegecko.min.css
```

The css are included like this in the generated html file:

```html
    <link rel="stylesheet" media="screen" href="pub/css/screen.css">
    <link rel="stylesheet" media="screen" href="pub/css/screen_overrides.css">
    <link rel="stylesheet" media="screen" href="pub/css/tradegecko.min.css">
    <link rel="stylesheet" media="print" href="pub/css/print.css">
    <link rel="stylesheet" media="print" href="pub/css/print_overrides.css">
    <link rel="stylesheet" media="print" href="pub/css/tradegecko.min.css">
    <link rel="stylesheet" media="screen" href="pub/css/darkula.css">
    <link rel="stylesheet" media="screen" href="pub/css/theme_overrides.css">
    <link rel="stylesheet" media="screen" href="pub/css/tradegecko.min.css">
```

In my opinion there are 2 issues here:

1. `theme_overrides.css` should be included last one. This way I can override the themes without having to modify inside their files. I guess this is the purpose of this file.
2. `tradegecko.min.css` is included twice in the `screen` context

Thanks,
Silviu

**Comment by MikeRalphson** [comment]
1. this ambiguity is due to the overloading of the term "theme". Here it refers to the syntax highlighting theme CSS, not the `--css` option "theme". However, I can't see any issue in re-ordering those.
2. Bug, now fixed.

**Comment by MikeRalphson** [comment]
Fixed in shins@2.3.2-6
